src/otp_tsv_to_upload.py

```diff
@@ -50,7 +50,7 @@ def load_file_metadata(otp_tsv: Path) -> list[FileMetadata]:
     with open(otp_tsv, "r", encoding="utf-8") as tsv_file:
         files = [
             FileMetadata(
-                path=Path(line.split("\t")[0].strip()),
+                path=Path(line.split("\t")[0].strip()).absolute(),
```

Comment:
```suggestion
                path=Path(line.split("\t")[0].strip()).resolve(),
```
Apparently absolute is now fine from Python 3.11 on, but undocumented + untested before... so your choice